Shopify/cli
# Import flow extensions command

### WHY are these changes introduced?
We need a new command to import dashboard-managed flow extensions into the CLI



### WHAT is this pull request doing?
- Add a new command `import-flow-legacy-extension`
  - This command uses the cached deploy app (or a `client-id` flag) to fetch the current flow extensions
  - The selected extensions are transformed into local files as `flow_action` and `flow_trigger`

- In a future PR: imported extensions will be migrated to be cli-managed.


### How to test your changes?
Having an app with dashboard-managed flow extensions, run:
- `pnpm shopify app import-flow-legacy-extensions --path {path} --client-id {client-id}`
- Select one or all extensions to be migrated
- Check that new folders and toml files are created in the extensions folder.



### Post-release steps



### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
